oak_functions/client/rust/src/attestation.rs

```diff
@@ -82,34 +81,42 @@ impl Client {
 /// gRPC Attestation Service client implementation.
 pub struct AttestationClient {
     client: Client,
-    encryptor: AeadEncryptor,
+    attested_state_machine: AttestationStateMachine<AttestedState>,
 }
 
 impl AttestationClient {
     pub async fn create(uri: &str, expected_tee_measurement: &[u8]) -> anyhow::Result<Self> {
         let mut client = Client::create(uri)
             .await
             .context("Couldn't create gRPC client")?;
-        let encryptor = AttestationClient::attest(&mut client, expected_tee_measurement)
-            .await
-            .context("Couldn't attest server")?;
+        let attested_state_machine =
+            AttestationClient::attest(&mut client, expected_tee_measurement)
```

Comment:
Could this be `Self::attest`? I think that reads better. And again, IIUC, `create_session` fits better.